materials_io/ase.py

```diff
@@ -0,0 +1,38 @@
+import json
+import ase
+from io import StringIO
+
+from materials_io.base import BaseSingleFileParser
+
+class AseParser(BaseSingleFileParser):
+    """Parse information from an input file using ASE.
```

Comment:
Can you be more explicit here? Mention that ASE is a parser for atomic simulation data and write about what kind of data it produces in more non-specialist-friendly terms